pkg/helper/client.go

```diff
@@ -185,10 +186,13 @@ func ApplyDesiredState(desiredState nmstatev1alpha1.State) (string, error) {
 		return "Ignoring empty desired state", nil
 	}
 
+	var setDoneCh = make(chan struct{})
+	go setUnavailableUp(setDoneCh)
```

Comment:
Can't this be done at fix-unavailable.go that implements Set and just run when needed ? (like checking we are at bonding and we are at kernel <= foobar)

Reply:
We don't know what exactly causes the issue. I managed to reproduce it on two different systems. I would prefer to run it all the time until we get to support a new enough platforms without this bug (maybe even 1.22, this however needs to get backported).